system-variables.md

```diff
@@ -4803,8 +4803,9 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 单位：字节
-- 默认值：`0`，自动设置内部统计信息缓存使用内存的上限为总内存的一半。
-- 这个变量用于控制 TiDB 内部统计信息缓存使用内存的上限。
+- 默认值：`0`，表示自动设置统计信息缓存的内存使用上限为总内存的一半。
+- 范围：`[0, 1099511627776]`
```

Comment:
I add the value range to align the description with that in English: https://docs.pingcap.com/tidb/v7.5/system-variables#tidb_stats_cache_mem_quota-new-in-v610 
<img width="928" alt="image" src="https://github.com/user-attachments/assets/0d38c804-3a9d-4ebc-8928-d0a38f93d263">

Reply:
Yes